resources/content_list.ts

```diff
@@ -133,10 +134,10 @@ const contentList: (ZoneIdType)[] = [
   ZoneId.TheBowlOfEmbers,
   ZoneId.TheNavel,
   ZoneId.TheHowlingEye,
-  ZoneId.CapeWestwind,
+  ZoneId.CapeWestwind, // changed to solo duty in 6.1
```

Comment:
I think we should remove it from here, as in general there's not support for solo duties.

Reply:
I wasn't sure where CN and KO are and if they need those to stick around for a bit.

Reply:
Ah, ok.  That's fair.  We can remove them eventually, then.